src/ui/common/src/components/tables/PaginatedTable.tsx

```diff
@@ -91,12 +91,23 @@ export const PaginatedTable: React.FC<PaginatedTableProps> = ({ data }) => {
                   >
                     {columns.map((column, columnIndex) => {
                       const value = row[column.name];
+                      let displayedValue = '';
+                      if (!!value) {
+                        if (
+                          column.type === 'json' ||
+                          column.type === 'object'
```

Comment:
Is `column.type === 'object'` because in `infer_schema`, we hardcoded the type to be `object`? Also, why do we need `column.type === 'json'` then?

Reply:
Also, I believe `json` is not used in practice since our normal table schema depends on pandas `dtype` which will make all python objects column to `object`